data/gui/themes/default/window/help_browser.cfg

```diff
@@ -150,28 +150,49 @@
 							[/column]
 
 							[column]
-								grow_factor = 1
-								border = "all"
-								border_size = 5
 								horizontal_alignment = "right"
-
-								[button]
-									id = "back"
-									definition = "left_arrow_ornate"
-								[/button]
+								
+								[grid]
```

Comment:
These shouldn't need to go in their own grid. Change the `left_arrow_ornate` column to `grow_factor = 0` and you should be able to add the toggle button at the same level as before.